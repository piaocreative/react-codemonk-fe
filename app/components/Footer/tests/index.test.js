import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

import FooterWrapper from '../index';

describe('<Footer />', () => {
  describe('Component: FooterWrapper', () => {
    it('should render', () => {
      const component = (
        <BrowserRouter>
          <FooterWrapper />
        </BrowserRouter>
      );
      const tree = renderer.create(component).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
