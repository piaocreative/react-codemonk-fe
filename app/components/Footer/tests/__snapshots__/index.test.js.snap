// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Component: FooterWrapper should render 1`] = `
<div
  className="footer-style__Footer-wiwfnq-1 hasqFC"
>
  <p
    className="P-tiusvc-0 eZYXtL p16"
  >
    <span>
      Â© CodeMonk 2022 - All rights reserved.
    </span>
  </p>
  <ul
    className="footer-links"
  >
    <li>
      <a
        href="https://www.codemonk.ai/terms-and-conditions/"
        target="_blank"
      >
        <span>
          Terms
        </span>
      </a>
    </li>
    <li>
      <a
        href="https://www.codemonk.ai/privacy-policy/"
        target="_blank"
      >
        <span>
          Privacy
        </span>
      </a>
    </li>
    <li>
      <a
        href="https://www.codemonk.ai/cookie-policy/"
        target="_blank"
      >
        <span>
          Cookie
        </span>
      </a>
    </li>
    <li>
      <a
        href="https://www.codemonk.ai/acceptable-use-policy/"
        target="_blank"
      >
        <span>
          Acceptable
        </span>
      </a>
    </li>
  </ul>
  <ul
    className="footer-style__SocialLinks-wiwfnq-0 iigCyK"
  >
    <li>
      <a
        href="https://www.linkedin.com/company/codemonk-ai/"
        target="_blank"
      />
    </li>
    <li>
      <a
        href="https://dribbble.com/CodeMonk_AI/about/"
        target="_blank"
      />
    </li>
    <li>
      <a
        href="https://www.facebook.com/codemonk.fow/"
        target="_blank"
      />
    </li>
  </ul>
</div>
`;
